Dockerfile

```diff
@@ -16,13 +16,13 @@ RUN apt-get update -qqy && DEBIAN_FRONTEND=noninteractive apt-get install -qqy -
     libmpfr6 \
 && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
 
-ARG VERSION=2.3.1
+ARG VERSION=2.4.3
 ARG NAME=MiniZincIDE-$VERSION-bundle-linux
 
 RUN cd /tmp \
     && curl -LOSs https://github.com/MiniZinc/MiniZincIDE/releases/download/$VERSION/$NAME-x86_64.tgz \
     && tar -xzf $NAME-x86_64.tgz \
-    && (cd $NAME \
+    && (cd $NAME-x86_64 \
```

Comment:
The final `rm -rf` does not actually delete the extracted directory anymore since the name has changed. Would it be better to append `-x86_64` to the `NAME`?

Reply:
Good catch; yes, it would indeed.
How should we proceed? I can make the change locally and squash the new commit to the previous one, but how should I push it to tjunttila/grading-minizinc so that it is ok for this pull request?

Reply:
You can make the change locally and squash the commits into one commit. Then you can force push into your fork (i.e., `push -f origin master`) so that the squashed history is updated in this pull request. I will then merge this PR.

Force pushing is needed since you change the history of your branch by squashing commits. Your Github fork would not accept a normal push. In this case, we want to change the history so that this small change is included in one commit.

Reply:
Should now be done. Thank you very much for the constructive and easy-to-follow instructions!